### PR TITLE
LAMBDA-1452-4

### PR DESCRIPTION
**Pull request by yahmed153**
update function.py to get traceID

### CODE REVIEW

**Comment by yahmed153** [comment]
FINALLY I CAN MERGE 😄

**Review by mrickard — APPROVED** [review]

**Comment by mrickard** [comment]
@yahmed153 Ah...looks like you'll need to sign your commits (which you could probably do via amend)

File: src/function.py [hunk 1]
```diff
@@ -495,14 +495,21 @@ def _package_log_payload(data):
...
+            trace_id = _get_trace_id(log_event["message"])
+
         log_message = {
             "message": log_event["message"],
             "timestamp": log_event["timestamp"],
             "attributes": {"aws": {}},
         }
 
+        if trace_id:
+            log_message["traceId"] = trace_id
```
Comment (kolanos) [inline src/function.py:511]: Shouldn't this be `trace.id`?

Comment (yahmed153) [reply src/function.py:511]: Do you mean former = `trace.id` vs latter = `traceId`. The example queries you posted are the same

Comment (yahmed153) [reply src/function.py:511]: I was basing it off the struct tag [nr-lmbda-extension](https://github.com/newrelic/newrelic-lambda-extension/blob/01dc845bfecd63d9c4038e0f3cbe6a5b69be9daa/telemetry/payload.go#L102 )

Comment (yahmed153) [reply src/function.py]: you are correct

Comment (kolanos) [reply src/function.py]: Whoops. Copy-pasted the same query twice. The second one should hve been `traceId`.

**Review by kolanos — APPROVED** [review]

**Pull request closed without merging**